<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIGN-IN SYSTEM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagesrc.qrc">
    <normaloff>:/image/batman.png</normaloff>:/image/batman.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginDialog{
border-image: url(:/image/sent.PNG);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>730</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
	border-bottom-color: rgb(255, 255, 255);
color:white;
	
	font: 14pt &quot;Arial&quot;;
font:bold;
border-radius : 13px;
	background-color: qlineargradient(spread:reflect, x1:0.213, y1:0.210409, x2:0.910112, y2:0.869, stop:0.106742 rgba(0, 72, 107, 255), stop:0.41573 rgba(61, 129, 162, 255), stop:0.719101 rgba(119, 180, 210, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
background-color: qlineargradient(spread:reflect, x1:0.213, y1:0.210409, x2:0.910112, y2:0.869, stop:0.106742 rgba(0, 72, 130, 220), stop:0.41573 rgba(61, 110, 180, 255), stop:0.719101 rgba(119, 200, 210, 255), stop:1 rgba(255, 215, 235, 225));
color:white;
border:none;
font:bold;
}</string>
   </property>
   <property name="text">
    <string>LOG IN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>730</y>
     <width>831</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
   font:bold;
   color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>220</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:2px solid transparent;
	font: 75 13pt &quot;Calibri&quot;;
font:bold;
background : transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>280</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	border:2px solid transparent;
	font: 75 13pt &quot;Calibri&quot;;
font:bold;
background : transparent;
}</string>
   </property>
   <item>
    <property name="text">
     <string>MALE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FEMALE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INTERSEX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRANS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERSONAL</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>350</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:2px solid transparent;
	font: 75 13pt &quot;Calibri&quot;;
font:bold;
background : transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>420</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:2px solid transparent;
	font: 75 13pt &quot;Calibri&quot;;
font:bold;
background : transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:2px solid transparent;
	font: 75 13pt &quot;Calibri&quot;;
font:bold;
background : transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>590</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
	border-bottom-color: rgb(255, 255, 255);
color:white;
	
	font: 14pt &quot;Arial&quot;;
font:bold;
border-radius : 13px;
	background-color: qlineargradient(spread:reflect, x1:0.213, y1:0.210409, x2:0.910112, y2:0.869, stop:0.106742 rgba(0, 72, 107, 255), stop:0.41573 rgba(61, 129, 162, 255), stop:0.719101 rgba(119, 180, 210, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
background-color: qlineargradient(spread:reflect, x1:0.213, y1:0.210409, x2:0.910112, y2:0.869, stop:0.106742 rgba(0, 72, 130, 220), stop:0.41573 rgba(61, 110, 180, 255), stop:0.719101 rgba(119, 200, 210, 255), stop:1 rgba(255, 215, 235, 225));
color:white;
border:none;
font:bold;
}</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>590</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
	border-bottom-color: rgb(255, 255, 255);
color:white;
	
	font: 14pt &quot;Arial&quot;;
font:bold;
border-radius : 13px;
	background-color: qlineargradient(spread:reflect, x1:0.213, y1:0.210409, x2:0.910112, y2:0.869, stop:0.106742 rgba(0, 72, 107, 255), stop:0.41573 rgba(61, 129, 162, 255), stop:0.719101 rgba(119, 180, 210, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
background-color: qlineargradient(spread:reflect, x1:0.213, y1:0.210409, x2:0.910112, y2:0.869, stop:0.106742 rgba(0, 72, 130, 220), stop:0.41573 rgba(61, 110, 180, 255), stop:0.719101 rgba(119, 200, 210, 255), stop:1 rgba(255, 215, 235, 225));
color:white;
border:none;
font:bold;
}</string>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>530</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
backround:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>45</y>
     <width>551</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label{
border-top-left-radius:100px;
border:15px solid transparent;
border-image: url(:/image/sent2.PNG);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2{
border:3px solid transparent;
border-image: url(:/image/eye.PNG);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,90);
border:5px solid transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Alpha'S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>DEDICATION, DETERMINATION AND DISCIPLINE !</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagesrc.qrc"/>
 </resources>
 <connections/>
</ui>
